docs/common/craft-parts/reference/plugins/npm_plugin.rst

```diff
@@ -0,0 +1,59 @@
+.. _craft_parts_npm_plugin:
+
+NPM Plugin
+=============
+
+The NPM plugin can be used for Node.js projects that use NPM (or Yarn) as the package manager.
+
+Keywords
+--------
+
+In addition to the common :ref:`plugin <part-properties-plugin>` and
+:ref:`sources <part-properties-sources>` keywords, this plugin provides the following
+plugin-specific keywords:
+
+npm-include-node
+~~~~~~~~~~~~~~~~~~
+**Type:** boolean
+**Default:** False
+
+If this option is set to ``true``, the plugin will download and include the 
+Node.js binaries and its dependencies in the Snap.
```

Comment:
```suggestion
Node.js binaries and its dependencies in the resulting package.
```

Since craft-parts doesn't only build snaps this is application neutral.